Libplanet/Blocks/InvalidBlockTotalDifficultyException.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace Libplanet.Blocks
+{
+    [Serializable]
+    public class InvalidBlockTotalDifficultyException : InvalidBlockException
+    {
+        public InvalidBlockTotalDifficultyException(string message)
```

Comment:
It would be better if it take `difficulty` and `totalDifficulty` and expose them as readonly properties.